install/image-references

```diff
@@ -10,7 +10,11 @@ spec:
     from:
       kind: DockerImage
       name: docker.io/openshift/origin-aws-machine-controllers:v4.0.0
+  - name: openstack-machine-controllers
+    from:
+      kind: DockerImage
+      name: docker.io/openshift/origin-openstack-machine-controllers:v4.0.0
   - name: libvirt-machine-controllers
     from:
       kind: DockerImage
-      name: docker.io/openshift/origin-libvirt-machine-controllers:v4.0.0
\ No newline at end of file
+      name: docker.io/openshift/origin-libvirt-machine-controllers:v4.0.0
```

Comment:
@enxebre @wking  Is it possible to use an image from the dev ImageStream here? The `openstack-machine-controllers` image is now being built by our CI system but it won't be published on docker hub for a while and the mirror to quay.io has not been set yet.

Reply:
Just reference registry.svc.ci.openshift.org/openshift/origin-v4.0:openstack-machine-controllers

That’s where the image will be pulled from anyway.

Reply:
a-ha, thanks a lot. I'll do that.